apps/admin_api/priv/swagger/admin/request_bodies.yaml

```diff
@@ -74,48 +74,98 @@ AdminGetBody:
           - id
         example:
           id: usr_01ce83q2zw7zk1dqr79t22zr1v
-InviteAcceptBody:
-  description: The parameters to use for accepting an invite
+
+AdminCreateBody:
+  description: The parameters to send to create an admin user
   required: true
   content:
     application/vnd.omisego.v1+json:
       schema:
         properties:
           email:
             type: string
-          token:
+          redirect_url:
             type: string
-          password:
+          full_name:
             type: string
-          password_confirmation:
+          calling_name:
             type: string
+          enabled: 
+            type: boolean
+          global_role: 
+            type: string
+          metadata:
+            type: object
+            additionalProperties: true
+          encrypted_metadata:
+            type: object
+            additionalProperties: true
         required:
           - email
-          - token
-          - password
-          - password_confirmation
+          - redirect_url
         example:
-          email: example@omise.co
-          token: dmWEOiEvlPfEpb2XPEkNkNYR4xEqNuf25E9hqBYwvzg
-          password: user_provided_password
-          password_confirmation: user_provided_password
+          email: user@example.com
+          redirect_url: 'https://domain/redirect_path?email={email}&token={token}'
```

Comment:
Can we have a "working link" (except base url) instead?
Something like `http://localhost:4000/admin/invite?email={email}&token={token}`